src/type-coercions.md

```diff
@@ -91,7 +95,7 @@ the block has a known type.
 
 Coercion is allowed between the following types:
 
-* `T` to `U` if `T` is a subtype of `U` (*reflexive case*)
+* `T` to `U` if `T` is a [subtype] of `U` (*reflexive case*)
```

Comment:
There seems to be an unanswered question here: https://github.com/rust-lang/reference/pull/342/files#r191050490

Reply:
I've had little experience with the term 'reflexive' in programming contexts, but in math, "is a subset of" is a reflexive relationship. I assume that "is a subtype of" would be the programming equivalent.

Reply:
Ah. I'm not super familiar with it, but since this is just adding a link, I won't hold up the PR on it.  AIUI, the reflexive subtype would be "T is a subtype of T", not U (in the math sense, T is a subset of T).

Reply:
I guess.. Subtyping is an example of a reflexive relationship, because types are subtypes of themselves. So.. That means that a subtype coersion is reflexive, because it could be coercing to Self. That sounds really finicky now that I say it.